numba/core/debuginfo.py

```diff
@@ -10,6 +10,15 @@
 from llvmlite import ir
 from numba.core import cgutils
 
+
+
+def _escape_unicode(x):
+    """Escape unicode chars used in LLVM debug metadata.
+    """
+    # We will simply replace them with '?' for now.
+    return x.encode('ascii', 'replace').decode()
```

Comment:
Note to self, gdb uses octal escape sequence: `\nnn` where `nnn` are octal repr of utf8-encoded code point.

e.g.

```
>>> list(map(oct, 'ø'.encode('utf8')))
['0o303', '0o270']
```
the escaped sequence is `\303\270`